msgpack/unpack.h

```diff
@@ -193,6 +194,21 @@ static inline int unpack_callback_map_item(unpack_user* u, unsigned int current,
         PyErr_Format(PyExc_ValueError, "%.100s is not allowed for map key", Py_TYPE(k)->tp_name);
         return -1;
     }
+    if (PyUnicode_CheckExact(k) || PyBytes_CheckExact(k)) {
```

Comment:
Should not make a big difference, but this can be combined with the previous condition in l.194 :)

Reply:
Thanks for the review! I fixed this.